mlserver/env.py

```diff
@@ -14,11 +15,14 @@
 
 def _extract_env(tarball_path: str, env_path: str) -> None:
     logger.info(f"Extracting environment tarball from {tarball_path}...")
-    with tarfile.open(tarball_path, "r") as tarball:
-        tarball.extractall(path=env_path)
+    try:
+        with tarfile.open(tarball_path, "r") as tarball:
+            tarball.extractall(path=env_path)
+    except Exception:
+        logger.info("Failed to extract environment tarball.")
```

Comment:
is this an `error` level instead of `info`? also perhaps log the actual exception / reason?

Reply:
I removed my changes since it's not related to the PR.